docs/content/doc/translation/guidelines.en-us.md

```diff
@@ -0,0 +1,15 @@
+---
+date: "2021-01-22T00:00:00+02:00"
+title: "Translation Guidelines"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "translation"
+    name: "Translation Guidelines"
+    weight: 70
+    identifier: "translation-guidelines"
+---
+
+It is used to provide a common set of rules to make sure the translation is consistent.
```

Comment:
```suggestion
This place is used to provide a common set of rules to make sure the translation is consistent.
```

I'd list all available translation guidelines (well, German only atm) here so people don't need to switch their language to find it.

Reply:
we could list langauges right ...